modules/machineset-gcp-enabling-gpu-support.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating_machinesets/creating-machineset-gcp.adoc
+
+:_content-type: PROCEDURE
+[id="machineset-gcp-enabling-gpu-support_{context}"]
+= Enabling GPU support for a machine set
+
+Google Cloud Platform (GCP) Compute Engine allows users to add GPUs to VM instances. Workloads that benefit from access to GPU resources can perform better on compute machines with this feature enabled. {product-title} on GCP supports NVIDIA GPU models in the A2 and N1 machine series.
+
+.Supported GPU configurations
+|====
+|Model name |GPU type |Machine types ^[1]^
+
+|NVIDIA A100
+|`nvidia-tesla-a100`
+a|* `a2-highgpu-1g`
+* `a2-highgpu-2g`
+* `a2-highgpu-4g`
+* `a2-highgpu-8g`
+* `a2-megagpu-16g`
+// Of these, only `a2-highgpu-1g` is mentioned in the test plan. Should we assume all of these types work, or not?
+
+|NVIDIA K80
+|`nvidia-tesla-k80`
+.5+a|* `n1-standard-1`
+* `n1-standard-2`
+* `n1-standard-4`
+* `n1-standard-8`
+* `n1-standard-16`
+* `n1-standard-32`
+* `n1-standard-64`
+* `n1-standard-96`
```

Comment:
I believe the `N1-high-memory` and `N1-high-cpu` machine types should be working as well. https://cloud.google.com/compute/docs/general-purpose-machines#n1_machines

Reply:
Ah, very good. Thanks!